### PR TITLE
Map fixes and other misc code stuff

### PR DESCRIPTION
**Pull request by RadiantFlash7**
Reduces RCD cost per use of mattercartridges.
Readds Fancy coffeeshop sign.
Re-adds Bluespace coffee.
Fixes bluespace coffee sprite and code.
Fixes access to rad shelter on sublevel 2,
Fixes space tiles in main level window of sec outpost,
Fixes SMES names and RCON names to be the same now.
Fixes Skydiving teleporter (Updated refractored code)
Removed extra APCs, and security cameras facing the wrong way.
Tries to fix Debris field teleporters (still don't understand why theres a second destination in the shuttle console. Halp.)
Increases explorer job slots. 2 is not 'nough.
Reduces efficiency of cooling loop for TEG. (Waiting on Izac's weakened TEGS for further update)
Buffs health of freezable airlocks, so spiders won't constantly destroy them so quick.
Fixes borked up wire connecting solars to main.
Fixes incorrect facing security camera in security.
Adds status displays to escape,
changes explorer limit from 5 to 3. (what was I thinking?)
Still an improvement over 2.

### CODE REVIEW

**Comment by RadiantFlash7** [comment]
Radiation vars are now undefined, and it's throwing up errors. I don't understand how my changes could've affected this?

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -353,9 +353,7 @@ var/global/list/##LIST_NAME = list();\
 #define RAD_LEVEL_HIGH 25
 #define RAD_LEVEL_VERY_HIGH 75
 
-// Calculation modes for effective radiation
```
Comment (izac112) [inline code/__defines/misc.dm]: 2 defines are gone

File: code/modules/mining/mine_turfs.dm [hunk 2]
```diff
@@ -6,6 +6,7 @@ var/list/mining_overlay_cache = list()
 	icon = 'icons/turf/walls.dmi'
 	icon_state = "rock-dark"
 	density = 1
+	opacity = 1
```
Comment (izac112) [inline code/modules/mining/mine_turfs.dm]: comment pls

File: code/game/objects/structures/signs.dm [hunk 3]
```diff
@@ -815,4 +815,10 @@
 /obj/structure/sign/poi/engineright
 	name = "I.C.V."
 	desc = "The charred name of a cargo ship of some description."
-	icon_state = "poi_engine2"
\ No newline at end of file
+	icon_state = "poi_engine2"
+
+/obj/structure/sign/coffee
```
Comment (izac112) [inline code/game/objects/structures/signs.dm]: //yw edit begin and end pls

Comment (RadiantFlash7) [reply code/game/objects/structures/signs.dm]: Just found out that we actually have a seperate file for new signs, that was apparently disabled.
Gonna move this there.

Comment (izac112) [reply code/game/objects/structures/signs.dm]: Sounds good to me

**Comment by RadiantFlash7** [comment]
Should be all set now. Sometimes I hate myself.

**Pull request merged**